vulnerabilities/importers/pysec.py

```diff
@@ -0,0 +1,178 @@
+
+    if "published" in raw_data:
+        date_published = datetime.strptime(
+            raw_data["published"][0:19], "%Y-%m-%dT%H:%M:%S"
+        ).replace(tzinfo=timezone.utc)
+    else:
+        date_published = None
+        logger.error("date_published not found " + raw_data["id"])
+
+    severity = []
+    if "severity" in raw_data:
+        for sever_list in raw_data["severity"]:
+            if "type" in sever_list and sever_list["type"] == "CVSS_V3":
+                severity.append(
+                    VulnerabilitySeverity(
+                        system=SCORING_SYSTEMS["cvssv3_vector"],
+                        value=sever_list["score"],
+                    )
+                )
+    elif "ecosystem_specific" in raw_data and "severity" in raw_data["ecosystem_specific"]:
+        severity.append(
+            VulnerabilitySeverity(
+                system=SCORING_SYSTEMS["generic_textual"],
+                value=raw_data["ecosystem_specific"]["severity"],
+            )
+        )
+    else:
+        severity = []
+        logger.warning(f"severity not found- {raw_data['id']!r}")
+
+    references = []
+    if "references" in raw_data:
+        references = [
+            Reference(url=ref["url"], severities=severity) for ref in raw_data["references"] if ref
+        ]
+
+    affected_package = []
+    if "affected" in raw_data:
+        for affected_pkg in raw_data["affected"]:
+            package = affected_pkg["package"]
+            if "ecosystem" in package and "name" in package:
+                purl = PackageURL(type=package["ecosystem"], name=package["name"])
+            else:
+                purl = ""
+                logger.error(f"purl affected_pkg not found - {raw_data['id'] !r}")
+
+            affected_version_range = None
+            if "versions" in affected_pkg:
+                affected_version_range = (
+                    # HINT:  Values larger than 1/3 of a buffer page cannot be indexed.
+                    PypiVersionRange(affected_pkg["versions"][slice(200)])
+                )
+
+            if "ranges" in affected_pkg:
+                for fix_range in affected_pkg["ranges"]:
+                    fixed_version = set()
+                    if "type" in fix_range:
+                        if fix_range["type"] == "ECOSYSTEM":
+                            filter_fixed = list(
+                                filter(lambda x: x.keys() == {"fixed"}, fix_range["events"])
+                            )
+                            list_fixed = [i["fixed"] for i in filter_fixed]
+                            for i in list_fixed:
+                                try:
+                                    fixed_version.add(PypiVersion(i))
+                                except InvalidVersion:
+                                    logger.error(
+                                        f"fixed_version InvalidVersion - PypiVersion - {raw_data['id'] !r}"
```

Comment:
There is not enough detail in this message to act on this:
What what the version? what was the InvalidVersion error message?